packages/issuer-sdk-js/assets/idos-credential-subject-v1.json

```diff
@@ -19,12 +19,14 @@
     "idDocumentFrontFile": "xsd:string",
     "idDocumentBackFile": "xsd:string",
     "selfieFile": "xsd:string",
-    "residentialAddress": "xsd:string",
+    "residentialAddressStreet": "xsd:string",
+    "residentialAddressHouseNumber": "xsd:string",
+    "residentialAddressAdditionalAddressInfo": "xsd:string",
+    "residentialAddressCity": "xsd:string",
+    "residentialAddressPostalCode": "xsd:string",
     "residentialAddressCountry": "aux:ISO_3166-1_alpha-2",
     "residentialAddressProofCategory": "xsd:string",
     "residentialAddressProofDateOfIssue": "xsd:date",
-    "residentialAddressProofFile": "xsd:string",
-    "issuanceDate": "xsd:date",
-    "expirationDate": "xsd:date"
```

Comment:
I'd leave these two in, since they're optional (everything is optional in json-ld unless defined otherwise).

Reply:
https://github.com/idos-network/idos-sdk-js/blob/db4cc390e176ae2a17c39fa1514d9eb2d1a9354e/packages/issuer-sdk-js/assets/v1.json#L40
https://github.com/idos-network/idos-sdk-js/blob/db4cc390e176ae2a17c39fa1514d9eb2d1a9354e/packages/issuer-sdk-js/assets/v1.json#L44

ExpirationDate and IssuanceDate are by default there, so this is redundant.

Reply:
Ah, prefect! Nevermind me!